src/Styleguide/Components/RecentlyViewed.tsx

```diff
@@ -89,3 +90,32 @@ export const RecentlyViewedFragmentContainer = createFragmentContainer(
     }
   `
 )
+
+export const RecentlyViewedQueryRenderer = () => {
+  return (
+    <Subscribe to={[AppState]}>
+      {({ state }) => {
+        return (
+          <QueryRenderer
+            environment={state.system.relayEnvironment}
```

Comment:
Via https://github.com/artsy/reaction/pull/1119/ this would be 

```jsx
export const RecentlyViewedQueryRenderer = () => {
  return (
    <Connect to={[AppState]}>
      {({ system: { relayEnvironment }}) => 
        <QueryRenderer
          environment={relayEnvironment}
          ...
        />
      })
   </Connect>
  )
}
```
```